recipes/pynng/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pynng" %}
+{% set version = "0.6.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pynng-{{ version }}.tar.gz
+  sha256: 40920554979b49241d98b483383e97885f681a78a24273d51e0f341da8c026f7
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [win]
```

Comment:
And the same here

Reply:
This one requires `cmake` and a `c-compiler` so it can not be no arch to my knowledge.

Reply:
Indeed! 🤦